keytableprinter/formatwarnings.go

```diff
@@ -77,7 +87,7 @@ func formatKeyWarningLines(warning status.KeyWarning) []string {
 	}
 }
 
-func countdownUntilExpiry(days uint, indent uint) string {
+func countdownUntilExpiry(days uint, indent int) string {
```

Comment:
Interesting change from `uint` to `int`, does this mean the function now supports negatives?

Reply:
I think it's better to do `int(len(...))` in the caller rather than changing the `uint` to `int` later. Because `countdownUntilExpiry` really does only deal with positive (unsigned) integers it seems wrong to hint that it can handle a signed (possibly negative) integer by accepting `int`

Reply:
Good shout! I'll address that.